src/services/services.ts

```diff
@@ -604,6 +604,12 @@ namespace ts {
             return;
         }
         return firstDefined(getAllSuperTypeNodes(classOrInterfaceDeclaration), superTypeNode => {
+            if (declaration.modifiers?.some(modifier => modifier.kind === SyntaxKind.StaticKeyword)) {

```

Comment:
I wanted to use `find`, but encountered an error.
```
src/services/services.ts:607:40 - error TS2339: Property 'find' does not exist on type 'ModifiersArray'.

607             if (declaration.modifiers?.find(modifier => modifier.kind === SyntaxKind.StaticKeyword)) {
                                           ~~~~
```

Reply:
`some` is more appropriate anyways, since you don't care which modifier it is, just that it exists